dbt/scripts/utils/helpers.py

```diff
@@ -28,14 +28,25 @@ def create_logger(
     if log_file_path is not None and os.path.exists(log_file_path):
         os.remove(log_file_path)
 
-    # Create and start the default stream logger
-    logging.basicConfig(
-        level=logging.INFO,
-        format="%(asctime)s - %(levelname)s - %(message)s",
-        datefmt="%Y-%m-%d_%H:%M:%S.%f",
-        force=True,
+    formatter = logging.Formatter(
+        fmt="%(asctime)s.%(msecs)03d - %(levelname)s - %(message)s",
+        datefmt="%Y-%m-%d_%H:%M:%S",
```

Comment:
Note a slight difference in the datetime format here: Python's logging module uses `strftime` under the hood, which does not support the `%f` microseconds directive. I pulled this alternative from [this SO answer](https://stackoverflow.com/a/31335874).